packages/core/src/components/dropdown/test/default/dropdown.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/dropdown/test/default/index.html';
+
+test.describe('tds-dropdown-default', () => {
+  test('renders default dropdown correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dropdown = page.locator('tds-dropdown');
+    expect(dropdown).toBeTruthy();
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('should have a label outside with text', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const labelOutside = page.locator('tds-dropdown > div[class="label-outside"]');
```

Comment:
If we refactor the dropdown in the future and afterwards the div inside doesn't exist or the class "label-outside" is called something else, for example, the test will break. 
Unfortunately tests like these will be in the way for future work. 

I can recommend this article on the topic https://kentcdodds.com/blog/testing-implementation-details

Reply:
Playwright also mentions this here https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors 
getByRole seems like a good one, but it requires that the component have been implemented in an accessible way, which is perhaps not the case a lot of the time in this code base unfortunately.

Reply:
Sometimes, screenshots might be the only way we can test something in a resilient way.

Reply:
I am sorry, should have been clearer about this before

Reply:
Hehe no problem ma dude :) @alexandereneroth Sounds like a really nice article and I think this is what I feel when I do these kind of tests. "There must be a better way" but then I just bite and try to test the specs from the excel sheet. I think we have to do another think around these with a lot more tries to see what fits us. I agree totally the tests will break if something changes and we need to other checks. Screenshots are so good tests!